cygnus/1.0.0/mastermind.yml

```diff
@@ -1,6 +1,36 @@
-# TODO: Fill in properly
 name: cygnus
 version: 1.0.0
-description: Cygnus
+description: Cygnus is a connector in charge of persisting Orion Context Broker context data in certain configured third-party storages, creating a historical view of such data.
 protocol_type: HTTP
 ngsi_version: 1
+environment_variables:
+  - variable: CYGNUS_MYSQL_HOST
+    name: MySQL Host
+    description: MySQL Host to use. It should be mysql in this recipe.
```

Comment:
rather than this, say it should be equal to the name of the mysql service or something like that (i.e, expose better the reason)